README.md

```diff
@@ -81,13 +81,15 @@ require "stripe"
 Stripe::Charge.list(
   {},
   :api_key => "sk_test_...",
-  :stripe_account => "acct_..."
+  :stripe_account => "acct_...",
+  :stripe_version => "2018-02-28"
```

Comment:
shouldn't this be in its own hash? Like I know it works but we usually recommend having 2 separate hashes right? If so, we need to change the example below to be `id + {} + {xxxx}`a

Reply:
Yeah, it's a little cleaner as two hashes. I've updated both examples.

I'm not sure I understood what you meant by `id + {} + {xxxx}` though — would you mind taking another look to make sure this is what you had in mind? The signature for retrieve should just be `retrieve(id, opts = {})`.